### PR TITLE
Change location of DockerFiles with dependencies for Kanister Examples Helm release

### PR DESCRIPTION
**Pull request by rahulchheda**
Change location of DockerFiles with dependencies for Kanister Examples Helm release

Signed-off-by: Rahul M Chheda <rchheda@infracloud.io>

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [x] :robot: Test

## Issues

- #XXX

## Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

Changes Made:
- Changed docker repo form kanisterio to rahulchheda1997in .goreleaser.yaml
- Made fake release by running this command: goreleaser release --skip-publish --snapshot --rm-dist
- Logs for above release: https://gist.github.com/rahulchheda/16840d69fd225015e248ae1748084a8a/raw/da31976755d24b3946da6f1a6ce8d51f3e035ff6/go_releaser_logs.txt
- Pushed all images to my personal docker repo
- Image change for kanister operator pod. ref: https://gist.github.com/rahulchheda/16840d69fd225015e248ae1748084a8a/raw/da31976755d24b3946da6f1a6ce8d51f3e035ff6/pod_image.txt
- Made changes to all test files to use these pushed images. Changes ref: https://gist.github.com/rahulchheda/16840d69fd225015e248ae1748084a8a
- Ran make integration-test to verify everything is working fine. Logs: https://gist.github.com/rahulchheda/16840d69fd225015e248ae1748084a8a/raw/da31976755d24b3946da6f1a6ce8d51f3e035ff6/make_integration_test.txt

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -67,26 +67,26 @@ dockers:
...
+  dockerfile: 'docker/kanister-elasticsearch/image/Dockerfile'
   extra_files:
-  - 'examples/helm/kanister/kanister-elasticsearch/image/esdump-setup.sh'
+  - 'docker/kanister-elasticsearch/image/esdump-setup.sh'
 - binaries:
   - kando
   image_templates:
   - 'kanisterio/mysql-sidecar:{{ .Tag }}'
-  dockerfile: 'examples/helm/kanister/kanister-mysql/image/Dockerfile'
+  dockerfile: 'docker/kanister-mysql/image/Dockerfile'
```
Comment (viveksinghggits) [inline .goreleaser.yml:77]: I was wondering if all the `Dockerfile`s that we are moving from `examples` to `docker` are actually being used somewhere.

File: docker/kanister-mongodb-replicaset/image/LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+
+   1. Definitions.
```
Comment (viveksinghggits) [inline docker/kanister-mongodb-replicaset/image/LICENSE:7]: I dont think there is a need to manage this file separately.

File: .goreleaser.yml (hunk 1 above)
Comment (rahulchheda) [reply .goreleaser.yml:77]: So, the purpose was to move the docker files that are being used in releaser to `docker` dir, so yes, they are being used

**Comment by viveksinghggits** [comment]
LGTM

**Pull request merged**